lib/Dialect/SV/Ops.cpp

```diff
@@ -9,6 +9,11 @@
 using namespace circt;
 using namespace sv;
 
+/// Return true if the specified operation is an expression.
+bool sv::isExpression(Operation *op) {
```

Comment:
I guess this could actually be a trait on the ops, which EmitVerilog looks for. I'm not sure if it is worth adding just yet, but calling it out since we have had conversations about using some sort of interfaces for System Verilog emission before.

Reply:
That seems like an incremental way to begin abstracting EmitVerilog, especially since it would presumably include all the ops in `EmitVerilog::isVerilogExpression()` below.